example/images/migrations/0002_alter_customimage_file_hash.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.14 on 2022-07-25 13:13
```

Comment:
normally we keep the example app migrations to the min supported Wagtail version.
I double checked and with 2.15 only the blogpage body migration gets generated.